Vagrantfile

```diff
@@ -6,6 +6,7 @@ ENV['VAGRANT_NO_PARALLEL'] = 'yes'
 ENV['VAGRANT_EXPERIMENTAL'] = 'typed_triggers,disks'
 
 number_of_nodes = 3
+vagrant_nat_network = '10.1.0.0'
```

Comment:
instead of calling it nat, can you rename it to `vagrant_management_network`? this is to align the name to its role, which is serving as the vagrant management network (its also known as that by vagrant-libvirt at https://github.com/vagrant-libvirt/vagrant-libvirt/blob/9e7d0f3ec8c7b9c673672b801ec58e5926c08dfb/docs/configuration.markdown?plain=1#L564).

after that, I'll have to see how to accomplish the same with the other hypervisors (libvirt/kvm and hyper-v).

Reply:
done !